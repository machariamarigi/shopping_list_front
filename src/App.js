import React from 'react';
import { Provider } from 'react-redux';
import { BrowserRouter, Switch, Route } from 'react-router-dom';

import './App.css';
import store from './store';
import Landing from './containers/Landing'


const App = () => (
  <Provider store={store}>
    <BrowserRouter>
      <Switch>
        <Route path="/" component={Landing} />
      </Switch>
    </BrowserRouter>
  </Provider>
);

export default App;
