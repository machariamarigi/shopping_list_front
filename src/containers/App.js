import React from 'react';

import './App.css';
import Landing from './Landing/Landing';


const App = () => <div className="App"><Landing /></div>;

export default App;
