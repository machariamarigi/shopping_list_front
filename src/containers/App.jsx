import React, { Component } from 'react';
import logo from '../images/logo.svg';
import './App.css';


class App extends Component {
  render() {
    return (
      <div className="App">
        App Home
      </div>
    )
  }
}

export default App;
