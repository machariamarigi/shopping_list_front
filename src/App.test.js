import React from 'react';
import App from './App';

const localStorageMock = {
  getItem: jest.fn(),
  setItem: jest.fn(),
  clear: jest.fn(),
};
global.localStorage = localStorageMock;

it('renders without crashing', () => {
  shallow(<App />);
});
