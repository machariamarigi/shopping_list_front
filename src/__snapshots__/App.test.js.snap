// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App snapshots 1`] = `
<div>
  <div
    className={undefined}
    style={
      Object {
        "WebkitTapHighlightColor": "rgba(0,0,0,0)",
        "backgroundColor": "#00bcd4",
        "borderRadius": "0px",
        "boxShadow": "0 1px 6px rgba(0, 0, 0, 0.12),
               0 1px 4px rgba(0, 0, 0, 0.12)",
        "boxSizing": "border-box",
        "color": "rgba(0, 0, 0, 0.87)",
        "display": "flex",
        "fontFamily": "Roboto, sans-serif",
        "muiPrepared": true,
        "paddingLeft": 24,
        "paddingRight": 24,
        "position": "relative",
        "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
        "width": "100%",
        "zIndex": 1100,
      }
    }
  >
    <h1
      onClick={[Function]}
      style={
        Object {
          "boxFlex": 1,
          "color": "#ffffff",
          "flex": "1",
          "fontSize": 24,
          "fontWeight": 400,
          "height": 64,
          "letterSpacing": 0,
          "lineHeight": "64px",
          "margin": 0,
          "muiPrepared": true,
          "overflow": "hidden",
          "paddingTop": 0,
          "textOverflow": "ellipsis",
          "whiteSpace": "nowrap",
        }
      }
    >
      Shopping List App
    </h1>
    <div
      style={
        Object {
          "marginLeft": "auto",
          "marginRight": -16,
          "marginTop": 8,
          "muiPrepared": true,
        }
      }
    >
      <div>
        <a
          href="/login"
          onClick={[Function]}
        >
          <button
            disabled={false}
            onBlur={[Function]}
            onClick={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            onMouseEnter={[Function]}
            onMouseLeave={[Function]}
            onTouchStart={[Function]}
            style={
              Object {
                "WebkitTapHighlightColor": "rgba(0, 0, 0, 0)",
                "backgroundColor": "rgba(0, 0, 0, 0)",
                "border": 10,
                "borderRadius": 2,
                "boxSizing": "border-box",
                "color": "rgba(0, 0, 0, 0.87)",
                "cursor": "pointer",
                "display": "inline-block",
                "fontFamily": "Roboto, sans-serif",
                "fontSize": "inherit",
                "fontWeight": "inherit",
                "height": 36,
                "lineHeight": "36px",
                "margin": 0,
                "minWidth": 88,
                "muiPrepared": true,
                "outline": "none",
                "overflow": "hidden",
                "padding": 0,
                "position": "relative",
                "textAlign": "center",
                "textDecoration": "none",
                "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
                "userSelect": "none",
                "verticalAlign": null,
              }
            }
            tabIndex={0}
            type="button"
          >
            <div
              onMouseDown={[Function]}
              onMouseLeave={[Function]}
              onMouseUp={[Function]}
              onTouchEnd={[Function]}
              onTouchStart={[Function]}
            >
              <span
                style={
                  Object {
                    "fontSize": 14,
                    "fontWeight": 500,
                    "letterSpacing": 0,
                    "muiPrepared": true,
                    "paddingLeft": 16,
                    "paddingRight": 16,
                    "position": "relative",
                    "textTransform": "uppercase",
                    "verticalAlign": "middle",
                  }
                }
              >
                Login
              </span>
            </div>
          </button>
        </a>
        <a
          href="/register"
          onClick={[Function]}
        >
          <button
            disabled={false}
            onBlur={[Function]}
            onClick={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            onMouseEnter={[Function]}
            onMouseLeave={[Function]}
            onTouchStart={[Function]}
            style={
              Object {
                "WebkitTapHighlightColor": "rgba(0, 0, 0, 0)",
                "backgroundColor": "rgba(0, 0, 0, 0)",
                "border": 10,
                "borderRadius": 2,
                "boxSizing": "border-box",
                "color": "rgba(0, 0, 0, 0.87)",
                "cursor": "pointer",
                "display": "inline-block",
                "fontFamily": "Roboto, sans-serif",
                "fontSize": "inherit",
                "fontWeight": "inherit",
                "height": 36,
                "lineHeight": "36px",
                "margin": 0,
                "minWidth": 88,
                "muiPrepared": true,
                "outline": "none",
                "overflow": "hidden",
                "padding": 0,
                "position": "relative",
                "textAlign": "center",
                "textDecoration": "none",
                "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
                "userSelect": "none",
                "verticalAlign": null,
              }
            }
            tabIndex={0}
            type="button"
          >
            <div
              onMouseDown={[Function]}
              onMouseLeave={[Function]}
              onMouseUp={[Function]}
              onTouchEnd={[Function]}
              onTouchStart={[Function]}
            >
              <span
                style={
                  Object {
                    "fontSize": 14,
                    "fontWeight": 500,
                    "letterSpacing": 0,
                    "muiPrepared": true,
                    "paddingLeft": 16,
                    "paddingRight": 16,
                    "position": "relative",
                    "textTransform": "uppercase",
                    "verticalAlign": "middle",
                  }
                }
              >
                Register
              </span>
            </div>
          </button>
        </a>
      </div>
    </div>
  </div>
  <div
    className="container"
  >
    <div
      style={
        Object {
          "WebkitTapHighlightColor": "rgba(0,0,0,0)",
          "backgroundColor": "#ffffff",
          "borderRadius": 2,
          "boxShadow": "0 1px 6px rgba(0, 0, 0, 0.12),
                 0 1px 4px rgba(0, 0, 0, 0.12)",
          "boxSizing": "border-box",
          "color": "rgba(0, 0, 0, 0.87)",
          "fontFamily": "Roboto, sans-serif",
          "muiPrepared": true,
          "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
          "zIndex": 1,
        }
      }
    >
      <div
        style={
          Object {
            "paddingBottom": 0,
          }
        }
      >
        <div
          style={
            Object {
              "muiPrepared": true,
              "position": "relative",
            }
          }
        >
          <div
            style={
              Object {
                "muiPrepared": true,
              }
            }
          >
            <img
              alt=""
              src="https://www.buzzfeed.com/static-assets/img/shopping_header.bd16ddeece64859c9f790c30a332a0ed.png"
              style={
                Object {
                  "maxWidth": "100%",
                  "minWidth": "100%",
                  "muiPrepared": true,
                  "verticalAlign": "top",
                  "width": "100%",
                }
              }
            />
          </div>
          
        </div>
      </div>
    </div>
  </div>
</div>
`;
